app/screens/about/index.js

```diff
@@ -1,24 +1,27 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {connect} from 'react-redux';
+import {realmConnect} from 'realm-react-redux';
 
-import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
-import {getConfig, getLicense} from 'mattermost-redux/selectors/entities/general';
-import {isLandscape} from 'app/selectors/device';
+import {General, Preferences} from 'app/constants';
+import {getTheme} from 'app/realm/selectors/preference';
+import options from 'app/store/realm_options';
 
 import About from './about';
 
-function mapStateToProps(state) {
-    const config = getConfig(state);
-    const license = getLicense(state);
+function mapPropsToQueries(realm) {
+    const general = realm.objectForPrimaryKey('General', General.REALM_SCHEMA_ID) || General.REALM_EMPTY_OBJECT;
+    const themePreferences = realm.objects('Preference').filtered('category = $0', Preferences.CATEGORY_THEME);
 
+    return [general, themePreferences];
+}
+
+function mapQueriesToProps([general, themePreferences]) {
     return {
-        config,
-        license,
-        theme: getTheme(state),
-        isLandscape: isLandscape(state),
+        config: general?.config || JSON.parse(general?.serverConfig),
```

Comment:
Why the JSON.parse?

Reply:
The serverConfig is being returned as a string rather than an object and this is to break it back out into an object.